doc/source/index.rst

```diff
@@ -9,10 +9,6 @@ Ansys Sphinx theme documentation |version|
    :hidden:
    :maxdepth: 3
 
-   installation
-   css
-   options
-   404_page
-
-
+   Getting_started/index.rst
```

Comment:
Rename the ``Getting_started`` folder to ``getting_started`` and adapt where needed
```suggestion
   getting_started/index.rst
```